app/views/defence_requests/show.html.erb

```diff
@@ -14,7 +14,7 @@
 <% end %>
 
 <dl class="labels">
-  <%= display_value "gender", @defence_request.gender %>
+  <%= display_value "gender", t(@defence_request.gender, default: t(:not_given)) %>
```

Comment:
Just FYI - I've changed the way this part of the detail is rendered - it will use a shared partial. So this will get overriden soon.